expected/01_general.out

```diff
@@ -351,3 +351,30 @@ SELECT 4, COUNT(*) = 0 FROM "PoWA".powa_statements_history;
         4 | t
 (1 row)
 
+-- Test toast_tuple_target: we shouldn't have any table belonging to powa archivist
+-- that has a column mins_in_range (it means it's a coalesced table) and isn't set
+-- for aggressive toasting
+WITH ext AS (
+    SELECT c.oid, c.relname, c.reloptions
+    FROM pg_depend d
+    JOIN pg_extension e ON d.refclassid = 'pg_extension'::regclass
+        AND e.oid = d.refobjid
+        AND e.extname = 'powa'
+    JOIN pg_class c ON d.classid = 'pg_class'::regclass
+        AND c.oid = d.objid
+    WHERE c.relkind != 'v'
+)
+SELECT ext.relname
+FROM ext
+WHERE EXISTS
+  (SELECT 1 FROM pg_attribute a
+   WHERE a.attrelid = ext.oid
+      AND a.attname ~ '(mins|maxs)'
```

Comment:
this should probably match the comment:
> that has a column mins_in_range

Alternatively update the comment but checking for exactly `mins_in_range` feels a bit safer, same for the actual function.

Reply:
Ok, fixed, but that was inspired by
```
-- check (mins|maxs)_in_range columns not marked as STORAGE MAIN
WITH ext AS (
    SELECT c.oid, c.relname
    FROM pg_depend d
    JOIN pg_extension e ON d.refclassid = 'pg_extension'::regclass
        AND e.oid = d.refobjid
        AND e.extname = 'powa'
    JOIN pg_class c ON d.classid = 'pg_class'::regclass
        AND c.oid = d.objid
    WHERE c.relkind != 'v'
)
SELECT ext.relname, a.attname
FROM ext
JOIN pg_attribute a ON a.attrelid = ext.oid
WHERE a.attname ~ '(mins|maxs)'
AND a.attstorage != 'm'
ORDER BY ext.relname::text COLLATE "C", a.attname::text COLLATe "C";
```
So maybe this one is wrong too in CI ?

Reply:
yep I totally botched this one.  I can fix it afterwards if needed.

Reply:
Yeah sure, it doesn't belong in this PR